### PR TITLE
tag 1.0.8, bower: no minify, include plugins

### PR DESCRIPTION
**Pull request by konsumer**
You can try it out with `bower install konsumer/prefixfree`

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -9,16 +9,15 @@
...
     "index.js",
     "minify",
-    "prefixfree.js"
+    "**/*.min.js"
   ],
   "description": "A script that lets you use only unprefixed CSS properties everywhere. It works behind the scenes, adding the current browser’s prefix to any CSS code, only when it’s needed.",
   "repository": {
     "type": "git",
-    "url": "git://github.com/LeaVerou/prefixfree.git"
+    "url": "git://github.com/konsumer/prefixfree.git#gh-pages"
```
Comment (LeaVerou) [inline bower.json]: I think that's leftover?

Comment (konsumer) [inline bower.json]: hehe, yeah, I just commented the same.

**Comment by konsumer** [comment]
resolves #45 & #177

File: bower.json (hunk 1 above)
Comment (LeaVerou) [inline bower.json]: Can you update the PR? Thanks btw!

Comment (konsumer) [inline bower.json]: done. no prob! great library.

**Comment by andyinabox** [comment]
+1!

**Comment by LeaVerou** [comment]
Thanks! Sorry for the delay!

**Pull request merged**